.github/workflows/macos-linux-windows-pixi.yml

```diff
@@ -87,8 +87,8 @@ jobs:
       - uses: actions/cache@v4
         with:
           path: .ccache
-          key: ccache-macos-linux-conda-${{ matrix.os }}-${{ matrix.build_type }}-${{ matrix.python-version }}-${{ github.sha }}
-          restore-keys: ccache-macos-linux-conda-${{ matrix.os }}-${{ matrix.build_type }}-${{ matrix.python-version }}-
+          key: ccache-macos-linux-conda-${{ matrix.os }}-${{ matrix.build_type }}-${{ matrix.environment }}-${{ github.sha }}
```

Comment:
This means the cache is conditional on the Pixi environment and not the Python version anymore. Is that what we want?

Reply:
In fact, python-version doesn't exists anymore.